apibindings/go/go.sum

```diff
@@ -0,0 +1,8 @@
+github.com/another-rex/osv.dev/go v0.4.0 h1:1UNsHVhM4KH5faHTXRHF4g1KyDDrrN477w+SxPKoKrM=
+github.com/another-rex/osv.dev/go v0.4.0/go.mod h1:Gb3fa+j3r4ewPodfnLlJRlWWKlML49rJmjLugRv/Wc8=
```

Comment:
Is this up to date? Should this be pointing to your fork?

Reply:
Ah needed to run go mod tidy to delete the outdated go sum lines